### PR TITLE
Bpk 1568 post neo cleanup part3

### PR DESCRIPTION
**Pull request by matteo-hertel**
This gets rid of all the Neo mentions, moves the neo components and makes them the default 

unfortunately is a humongous PR 😅 

I'll squash and whatnot if travis is happy and somebody casts an eye on it

### CODE REVIEW

File: packages/bpk-docs/src/components/DocsPageBuilder/DocsPageBuilder.js [hunk 1]
```diff
@@ -134,55 +153,75 @@ const CustomSection = section => [
...
-          [...props.components, ...props.customSections].map(
-            ExampleNavListItem,
-          ),
-        )}
-      </BpkList>
-    )}
-    {props.tokenMap ? <TokenSwitcher tokens={props.tokenMap} /> : null}
-    {props.usageTable
-      ? flatten([
+const NeoDocsPageBuilder = props => {
```
Comment (k0nserv) [inline packages/bpk-docs/src/components/DocsPageBuilder/DocsPageBuilder.js]: This shouldn't be called `NeoDocsPageBuilder` anymore should it?

File: packages/bpk-docs/src/pages/HomePageOld/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+/*
```
Comment (k0nserv) [inline packages/bpk-docs/src/pages/HomePageOld/HomePage.js]: Why do we need this?

File: packages/bpk-docs/src/components/DocsPageBuilder/DocsPageBuilder.js (hunk 1 above)
Comment (matteo-hertel) [reply packages/bpk-docs/src/components/DocsPageBuilder/DocsPageBuilder.js]: Ah I missed that, thanks!

File: packages/bpk-docs/src/pages/HomePageOld/HomePage.js (hunk 2 above)
Comment (matteo-hertel) [reply packages/bpk-docs/src/pages/HomePageOld/HomePage.js]: we don't, thanks for spotting it

**Pull request merged**